Add delete/default in TerminalFunctionality module

mnurzyns opened the pull request:
Every terminal command should be single, we do not need
to copy or move it, because we want only cast it.

mnurzyns committed:
Add delete/default in TerminalFunctionality module

Every terminal command should be single, we do not need
to copy or move it, because we want only cast it.

marcinwozniak0 commented on MessengerOnFileForLinux/TerminalFunctionality/src/EndConversation.hpp:
```diff
@@ -11,6 +11,11 @@ class EndConversation : public TerminalCommand
     EndConversation(std::string command, std::shared_ptr<ChatInformation> chatInfo);
     ~EndConversation();
```
default i wywalić z cpp

marcinwozniak0 commented on MessengerOnFileForLinux/TerminalFunctionality/src/HistoryDowloander.hpp:
```diff
@@ -12,6 +12,12 @@ class HistoryDowloander : public TerminalCommand
     HistoryDowloander(std::string command, std::string chatFileWithPath);
     ~HistoryDowloander() = default;
 
```
zbedna pusta linia

marcinwozniak0 commented on MessengerOnFileForLinux/TerminalFunctionality/src/InviteReceiver.hpp:
```diff
@@ -13,6 +13,11 @@ class InviteReceiver : public TerminalCommand
     InviteReceiver(std::string command, std::shared_ptr<ChatInformation> chatInfo);
     ~InviteReceiver();
```
default i wywalic z cpp

marcinwozniak0 commented on MessengerOnFileForLinux/TerminalFunctionality/src/InviteSender.hpp:
```diff
@@ -13,6 +13,11 @@ class InviteSender : public TerminalCommand
     InviteSender(std::string command, std::shared_ptr<ChatInformation> chatInfo);
     ~InviteSender();
```
default i wywalic z cpp

Commit:
Delete blank lines and correct missing default

Commit after review. There was missing default in two class
and too much blank lines in HistoryDowloander.
